sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/common/cloud-provider/fix.adoc

```diff
@@ -0,0 +1,73 @@
+== How to fix it
+
+To eliminate the security risks associated with leaked cloud provider secrets,
+it is important to list where this secret is used, who or what is using it, and
+carefully proceed with the following steps.
+
+=== Revoke the secret
+Once a secret access key is compromised or leaked, you must immediately revoke
+it, change it, and apply the changes to the entity using it.
+
+It is also important that the number of entities using this secret is as close
+to **1** as possible. If revoking and changing a secret results in multiple
+denials of service, this is another problem: passwords, secrets, and any type
+of credentials should only be used to authenticate a single entity (a person or
+a system).
+
+Revocation can be done via the cloud provider's web application or via its CLI
+and APIS.
+
+By revoking the secret access key, the compromised credentials are no longer
+valid, which immediately stops the movements of a particular intruder.
+
+**Important**: If a forensic analysis and incident response is underway, it is
+important to ask investigators before revoking secret access. If an attacker is
+currently lurking in the infrastructure, the unexpected revocation of their
+access will immediately tell them that they have been found and they could do
+their best to cover their tracks.
+
+=== Use a secret manager in backends
+After revoking a secret, it is important that you do not reinsert it into a
+plaintext file that is also stored in a public location. +
+Instead, use a dedicated secret management service provided by the cloud
+provider or your organization and follow its integration steps.
+
+These services enable centralized and secure storage of access keys and provide
+additional protection against unauthorized access and facilitate secure
+rotation of access keys.
+
+=== Using keystores in frontends
+
+When it comes to frontend applications and client-side environments, it is
+important to stop keeping the secret in code after revocation, even with
+obfuscation. Instead, use secure keystores or credential management systems
+provided by the programming language, framework, or environment in use. +
+These keystores can securely store access keys and ensure that they are
+protected from unauthorized access or accidental disclosure.
+
+=== Checking API Access for Malicious Activity
```

Comment:
The uppercasing is in a different style here than for the other headers.

Reply:
```suggestion
=== Checking API access for malicious activity
```